### ISSUE TITLE
Not having to re-click on 'draw a shape'

### ISSUE THREAD

**Issue by EliWei** [layersoflondon/application#582]
Booth map - you have to click on 'draw shape' every time you start a shape. Is this necessary? It would probably be easier if each shape autosaved when you completed the shape as many users were forgetting that you had to click draw shape and needed to be reminded.

This is a new feature. We estimated <2 days and Seif is fine to pay for it.

**Comment by EliWei** [comment]
Just to clarify, this is to be logged as LoL time in Nikabot

**Comment by andyError** [comment]
I don't think this needs any FED but gimme a shout if you want me to check the UX @phendrick .
